pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -125,4 +125,11 @@ reports:
 general:
   # Hostname that other components should use in order to connect to the Control Plane.
   # Control Plane will use this value in configuration generated for dataplanes, in responses to `kumactl`, etc.
-  advertisedHostname: localhost # ENV: KUMA_GENERAL_ADVERTISED_HOSTNAME
\ No newline at end of file
+  advertisedHostname: localhost # ENV: KUMA_GENERAL_ADVERTISED_HOSTNAME
+
+# Web GUI Server configuration
+guiServer:
+  # Port on which the server is exposed
+  port: 5683 # ENV: KUMA_GUI_SERVER_PORT
+  # Directory from which GUI is served. All files in this directory are exposed
+  directory: /path-to-kuma-cp/gui # ENV: KUMA_GUI_SERVER_DIRECTORY
```

Comment:
The primary use case for GUI server is to serve HTML resources embedded into `kuma-cp` binary.

I'm not sure if we should keep `Directory` field in the config. At the moment, it makes sense only to Kuma developers.